src/main/java/br/com/caelum/vraptor/test/http/Parameters.java

```diff
@@ -38,9 +39,17 @@ public void fill(MockHttpServletRequest request) {
 		}
 		for (Parameter param : parameters) {
 			request.addParameter(param.getName(),param.getValue());
-		}		
+		}
 		for (Parameter param : headers) {
 			request.addHeader(param.getName(), param.getValue());
-		}		
+		}
+		if(bytes != null && bytes.length > 0){
+			request.setContent(bytes);
+		}
+	}
+
+	public Parameters setContent(byte[] bytes) {
```

Comment:
i know, but we dont need to deal with byte[] directly. We can get it as string and than convert to byte here:
https://github.com/caelum/vraptor-test/pull/19/files/8337f64f94aa8663394b559c4faf1db0e99e9f84#diff-5be0cbdaaa72881e4a81c24e6eaecb38R47